.gitignore

```diff
@@ -12,6 +12,11 @@ local.properties
 # Visual Studio Code project files
 .vscode
 
+# Generated Docs
+/doc
+/modules
+/platform
```

Comment:
These shouldn't be needed. Using the `scripts/update_doc_classes_xml.sh` with Godot built from `master` shouldn't generate these directories - it should only update the `doc_classes/` directory (which we do want in the repo).

Or, is this coming from something else?